crypto/rand/rand_egd.c

```diff
@@ -72,49 +72,48 @@ _variable
 int hpns_socket(int family,
                 int type,
                 int protocol,
-                char* transport
-               )
+                char* transport)
 {
-  int  socket_rc;
-  char current_transport[20];
+    int  socket_rc;
+    char current_transport[20];
 
-#define AF_UNIX_PORTABILITY    "$ZAFN2"
-#define AF_UNIX_COMPATIBILITY  "$ZPLS"
+#  define AF_UNIX_PORTABILITY    "$ZAFN2"
+#  define AF_UNIX_COMPATIBILITY  "$ZPLS"
 
-  if (!_arg_present(transport) || !transport || transport[0] == '\0')
-  {
-    return socket(family,type,protocol);
-  }
+    if (!_arg_present(transport) || transport != NULL || transport[0] == '\0')
+    {
+      return socket(family, type, protocol);
```

Comment:
indent is still not right on the inner blocks :)

Reply:
Also for single block ifs remove the {}

Reply:
Argh. 3am. Sorry.

Reply:
Note: if this was a multi-statement block, the starting brace should be on the same line as the condition (strictly speaking, its end), some would have had to be this:

``` C
    if (!_arg_present(transport) || transport != NULL || transport[0] == '\0') {
        /* yada yada yada */
    }
```

Reply:
Understood. Same style used at my $DAYJOB.